### DISCUSSION TITLE
Show the toast inside other <div> element

### DISCUSSION THREAD

**Discussion by assapir** [fkhadra/react-toastify#1074] [Q&A]
Hey all.
I am starting to use `react-toastify`, which works like magic. The only thing I could not figure out is how to mount the toast (via `ToastContainer` I guess?) to show up inside a div, and not relative to the screen.

**Comment by ChrisBoon** [comment]
ToastContainer is positioned relative to the window using CSS - specifically `position:fixed`. You can see the CSS here: https://github.com/fkhadra/react-toastify/blob/main/scss/_toastContainer.scss

The how to style page of the docs shows several ways you could add your own CSS to overwrite it:
https://fkhadra.github.io/react-toastify/how-to-style

You can also use the minimal CSS instead and build out your on styles completely (https://fkhadra.github.io/react-toastify/how-to-style#styling-from-scratch) though that will require more effort.
